src/components/vistas/tablet/container-card/card/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import styles from './styles.module.scss';
+/*import Medicamento from '../../../../../assets/atripla.png'; en la linea 18 va este codigo <img src={Medicamento} alt='medicamento' className={styles.medicamento}/>*/
+import Bar from './qualitybar';
+
+class Card extends React.Component {
+    render(
+
+    ) {
+        return (
+            <div className={styles.Card}>
```

Comment:
Procurar mantener los estilos y el case (mayúsculas/minúsculas) consistente en el nombramiento de las clases.
```suggestion
            <div className={styles.card}>
```

Reply:
resuelto